### DISCUSSION TITLE
Pool payouts to cold wallet

### DISCUSSION THREAD

**Discussion by ghost** [Chia-Network/chia-blockchain#7205] [Q&A]
Hi all

Will this actually continue to work? I've just exchanged the xch_payout_address values in config.yaml with my cold wallet address.

Cheers

**Comment by gryan315** [comment]
I believe the hashes in the pool_list are pulled from the NFT on the blockchain, so changing them in your config does nothing? Someone please correct me if I am mistaken.

**Reply by loppefaaret** [reply]
nope, the wallet is sending them to the wallet that created the plot NFT - pools could offer additional code that lets its users redirect wins to other address
it will still work for the farmer reward 1/8 + fees

**Reply by ghost** [reply]
So the cold wallet is of no use in the pool setup (yet)?

**Comment by nizoo91** [comment]
I asked on Space Pool they simply told me to change the xch_payout_address in the config.yaml, I did but didn't work.

I am not sure why this wasn't properly implemented yet, I mean I am not winning like crazy, but I would hate to see one of the forks steal my Chia without me noticing when the Pool should have already implemented a workaround for this :/

**Comment by ghost** [comment]
There are pull requests, check https://github.com/Chia-Network/chia-blockchain-gui/pull/496 and https://github.com/Chia-Network/chia-blockchain/pull/7161

Looks like we soon can change the payout address in the NFT. Hope there will be some instructions as well.

**Reply by nizoo91** [reply]
That's awesome thanks for the links! 
Looks like the first link you provided got approved and committed to main yesterday!
